### PR TITLE
add pvcodes

### LINKED ISSUE #399: Pranjal Verma
Screenshot from 2021-03-19 15-10-09

### PR DESCRIPTION
**Pull request by pvcodes**
Hi, here is my first PR on EddieHubCommunity, looking forward for more of the.
resolve #399

### CODE REVIEW

**Review by schmelto — APPROVED** [review]

File: data.json [hunk 1]
```diff
@@ -682,5 +682,11 @@
     "githubUsername": "SubhamRaoniar28",
     "imageUrl": "https://user-images.githubusercontent.com/41485604/111068853-767f3e00-84f0-11eb-9122-80af0d842463.jpg",
     "issueNumber": 397
+  },
+  {
+    "name": "Pranjal Verma",
+    "githubUsername": "pvcodes",
+    "imageUrl": "https://avatars3.githubusercontent.com/pvcodes",
```
Comment (AllanRegush) [inline data.json:689]: This shouldn't be your user avatar it should be an image of your profile
```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/54075838/111760905-40153a80-88c5-11eb-9600-aa49a78ddd05.png",
```

**Review by AllanRegush — CHANGES_REQUESTED** [review]
Looking Good :+1: Just a small adjustment :nerd_face:

**Comment by pvcodes** [comment]
> Looking Good :+1: Just a small adjustment :nerd_face: 

Yup, I committed that too.

**Review by AllanRegush — APPROVED** [review]
Looks Good :+1:

**Comment by AllanRegush** [comment]
Looks like there is a conflict would you be able to fix it? If you need any help let us know :nerd_face:

**Comment by pvcodes** [comment]
Hi, @AllanRegush which conflict.

**Comment by AllanRegush** [comment]
Screenshot_20210322_082822

Here is some  [documantation](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-on-github) on resolving conflicts.

**Review by AllanRegush — APPROVED** [review]

**Pull request merged**